src/backup-devops.sh

```diff
@@ -45,6 +45,8 @@ while [[ $# -gt 0 ]]; do
       ;;
   esac
 done
+echo "=== Azure DevOps Repository Backup Script ==="
+echo "=== v.1.0.1 == 
```

Comment:
Consider using a variable for the version number. This way, you only need to update the version in one place. [medium]